crates/libeq_wld/src/parser/fragments/material.rs

```diff
@@ -93,14 +30,18 @@ pub struct MaterialFragment {
     // Bit 3 ........ Set to 1 if the texture is masked and semi-transparent.
     // Bit 4 ........ Set to 1 if the texture is masked but not semi-transparent.
     // Bit 31 ...... Apparently must be 1 if the texture isn’t transparent.
-    pub transparency_flags: TransparencyFlags,
+    pub render_method: RenderMethod,
```

Comment:
This actually looks wrong to me still. I think this field is _a_  `RenderMethod` but it's a different `RenderMethod` than the one used in `TwoDimensionalObjectFragment` 😿. I need to do a little bit of investigation here still.

Reply:
Some updates here. I'm beginning to think that the `RenderMethod` field may not be a bit field at all when the highest bit is set.

The ASCII material def: `RENDERMETHOD USERDEFINED 21` results in a value of `0x80000014`. Which is a `u32` with the highest bit set and the decimal number `20` in the low bits. This makes me think that when the render method is `USERDEFINED` the number provided is an identifier for a custom set of render options defined in the executable. Changing the userdefined value from 21 to 20 results in a value of 19 in the low bits. The fact that 1 is subtracted leads me to believe that this is meant to be an index, not a bit field.

That said when the highest bit is _not_ set then the RenderMethod flags seem to make sense (draw_style, shading, texture_style, etc.). So far I've seen this form only on `TwoDimensionalObjectFragment`s. That leads me to believe that these render methods are sort of a legacy option that end up being used for older fragment types and as the 3d features of this engine were developed they defined their own custom (i.e. user defined) render methods outside of the pre-defined ones.

That makes understanding how these things were rendered a bit more difficult but still possible. I'll need to collect all of the values used for this field to see how many user defined values there are. So far I've seen 1, 19, and 20.

Reply:
Oh and another thing that leads me to believe that this is the case. The flags listed by Windcatcher are a bit strange. For example bit 1 is thought to be masked, bit 2 semi-transparent but not masked, and bit 3 masked and semi-transparent.

First, if 1 and 2 are set they contradict one another. And second, if 1 means masked why would 3 also mean masked if this is a bit field?

Reply:
Great!  I agree those flags were really strange

Reply:
I went ahead and made `RenderMethod` an enum covering the `Standard` and `UserDefined` cases.